xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp

```diff
@@ -261,19 +284,31 @@ bool CActiveAEBufferPoolResample::Create(unsigned int totaltime, bool remap, boo
 
   if (m_useResampler || m_changeResampler)
   {
-    m_resampler = CAEResampleFactory::Create();
+    if(!m_resampler)
```

Comment:
I mentioned this during a debugging session from adsp.
It's the same as above. AE has no feedback from adsp.